ofrak_core/ofrak/model/viewable_tag_model.py

```diff
@@ -16,19 +16,37 @@
 
 _VIEW_ATTRIBUTES_TYPE = "__view_attributes_type__"
 _COMPOSED_ATTRIBUTES_TYPE = "__composed_attributes_types__"
-_VIEW_ATTRIBUTE_TYPE_NAME_SUFFIX = "AutoAttributes"
 
 
 RA = TypeVar("RA", bound=ResourceAttributes)
 RVI = TypeVar("RVI", bound="ResourceViewInterface")
 
 
 class AttributesType(ResourceAttributes, Generic[RVI]):
+    """
+    A Generic type for `ViewableResourceTag` to get the auto-generated `ResourceAttributes`
+    class associated with that view type. The returned class is a `dataclass` which encapsulates
+    the fields defined in one specific `ViewableResourceTag`.
+    For example if `B` inherits from `A` and `A` defines one or more new fields,
+    `AttributesType[B]` has only fields defined in `B`, and none of the fields defined in `A`.
```

Comment:
If these are separate paragraphs, they need a blank line between to be separated in rendered Markdown. If they're not supposed to be separate paragraphs, the lines can be combined and re-wrapped.

```suggestion
    A Generic type for `ViewableResourceTag` to get the auto-generated `ResourceAttributes`
    class associated with that view type. The returned class is a `dataclass` which encapsulates
    the fields defined in one specific `ViewableResourceTag`.
    
    For example if `B` inherits from `A` and `A` defines one or more new fields,
    `AttributesType[B]` has only fields defined in `B`, and none of the fields defined in `A`.
```